### PR TITLE
Hack around USI Core being in too many places.

### PR DESCRIPTION
**Pull request by bobprime**
The metadata for RoverDude's projects no longer matches his current project configuration.  Specifically his USI-Core project is included in multiple other projects.  The fixes the ones that are currently in ckan.

It adds the USI-Core to ckan.  Makes both UKS and Karbonite depend on it, and then filters the USI-Core files back out of both of those .netkan files.

When people want to add MKS-Lite and Orion all they will have to do is create netkan files that depend on USI-Core and add a filter 
 "filter"     : ["USICore.version", "CHANGELOG.txt", "Kontainers", "ReactorPack"],
.

I have tested this on my install and the final outcome is the same as manually unpacking the zipfiles.  I also tested in game that all of the parts are there, and they are.

### CODE REVIEW

File: NetKAN/Karbonite.netkan [hunk 1]
```diff
@@ -26,7 +27,7 @@
         {
             "file"       : "GameData/UmbraSpaceIndustries",
```
Comment (plague006) [inline NetKAN/Karbonite.netkan]: "find": "UmbraSpaceIndustries/Karbonite",
"install_to" : "GameData/UmbraSpaceIndustries"

No filter required (means that changes to USI-Core will not break the Karbonite install.

File: NetKAN/UKS.netkan [hunk 2]
```diff
@@ -25,7 +25,9 @@
   "install" : [
     {
       "file"       : "GameData/UmbraSpaceIndustries",
```
Comment (plague006) [inline NetKAN/UKS.netkan]: Apply the same logic as the Karbonite comment (atm not able to quickly look at the contents of UKS.zip)

**Comment by plague006** [comment]
How will these changes affect RoverDude's other mods? `USI-SRV` `USI-LS` `USI-FTT` `USI-EXP` `USI-ART` `SoundingRockets` `AlcubierreStandalone`

**Comment by bobprime** [comment]
The only two mods on ckan right now that have the USI Core included in their distribution are UKS and Karbonite.  The core just contains reactors and container elements.  It will impact MKS-Lite and Orion when they get added to ckan because they both use those and will be able to use the same fix.

I have tested it with SRV, LS, FTT, and, EXP.  I just checked ART and Alcubirre and they do not contain the broken file.

The find for UKS will need to be for UmbraSpaceIndustries/Kolonization .  Would you like me to make those changes and issue another pull request (I am fairly new to GitHub).  Or does just pushing to my branch update the pull?

**Comment by plague006** [comment]
Pushing to a branch updates the PR.

**Pull request closed without merging**